pcb/rules/rule.py

```diff
@@ -26,28 +26,7 @@ def mmToMicrons(mm):
 
 def getStartPoint(graph):
     if 'center' in graph:
-        return graph['end']
-    elif 'angle' in graph:
-        # dosome magic to find the actual start point
-        # fetch values
-        x_c = graph['start']['x']
-        y_c = graph['start']['y']
-        x_e = graph['end']['x']
-        y_e = graph['end']['y']
-        a_arc = graph['angle']
-        # calculate radius
-        dx = x_c - x_e
-        dy = y_c - y_e
-        r = math.hypot(dx, dy)
-        # calculate vector of length 1 of the end point
-        dx_s = dx / r
-        dy_s = dy / r
-        # now get the angle of the end point
-        a = math.degrees(math.atan2(dy_s, dx_s))
-        a_s = math.radians(a + a_arc)
-        x_s = x_c - math.cos(a_s) * r
-        y_s = y_c - math.sin(a_s) * r
-        return {'x': x_s, 'y': y_s}
```

Comment:
Was this on purpose or is this a merge artefact?

Reply:
Pretty sure it should be a merge artifact. I did not deliberately touch that.

Reply:
So could you revert that?

Reply:
Odd that occurred. I thought the upstream/master also had this deleted. But just checked and it should in fact be there. I hope to get some time within the next few days to fix it.